tests/test_transformation/test_boxcox_transformer.py

```diff
@@ -36,16 +36,16 @@ def test_automatically_finds_variables(df_vartypes):
 
 def test_fit_raises_error_if_df_contains_na(df_na):
     # test case 2: when dataset contains na, fit method
+    transformer = BoxCoxTransformer()
```

Comment:
In fact, I think these are the changes I made in the PR I mention previously. So there is probably a problem with rebase.

Could you try syncing your main branch with the repo's main branch, and then rebasing to this branch so these files disappear?

Reply:
fixed with rebase